### DISCUSSION TITLE
Using spring JDBC or reading data from my DB on Karavan

### DISCUSSION THREAD

**Discussion by MN-Dheeraj** [apache/camel-karavan#1243] [Q&A]
### Ask a question

How to use spring JDBC or read data from my DB on Karavan? Is there any document that explains this?

### Variant

Web Application

### Container Management (if applicable)

Docker

### Operating System (if applicable)

Windows

### Version

4.4.0

**Comment by kondetiramanakumar** [comment]
@davsclaus @mgubaidullin could you please provide your inputs on this????????

**Comment by davsclaus** [comment]
there is a database example here
ttps://github.com/KaotoIO/kaoto-examples/tree/main/postgresql-to-log

**Reply by MN-Dheeraj** [reply]
@davsclaus , Thanks for sharing this.
However, my query is bit different. Let me elaborate.

I have an oracle database that I have to link to my karavan by adding all the properties. 
I created a bean 'dataSource' with all the properties of my database. I am trying to inject that into my java code but it doesn't work.

Or I would atleast need another way to utilize this data source by running sql queries manually with the support of component. In case, there is a component, I need to know what input takes. This is because I used 'poll enrich' component and but saw an issue. I couldn't probably understand how it had to be used.

Best Regards,
MN. Dheeraj

**Comment by davsclaus** [comment]
Not much docs but some examples like I pointed out.

Talisman also have some docs at
https://docs.talismanplatform.com/user-guide/basics/create-bean/
